### PR TITLE
Fix for default-function

### LINKED ISSUE #2: reexporting a default function turns into const with type any
# Bug Report

Moved from Piral.

## Prerequisites

- [x] Can you reproduce the problem in a [MWE](https://en.wikipedia.org/wiki/Minimal_working_example)?
- [x] Are you running the latest version?
- [x] Did you perform a search in the issues?

## Environment Details and Version

piral 0.10.9

## Description

When you default export a function, then **export this in a named fashion** in your exports.ts file, it becomes a const with type any.

## Steps to Reproduce

1. first.ts: export default function
2. second.ts: import default under a name
3. second.ts: export the now named function
4. api.ts

## Expected behavior

Should export the function type

## Actual behavior

Exports a const with type any.

## Possible Origin / Solution

See the MWE: https://github.com/SantoJambit/shell-mwe/commit/b534b409730ba00bded5b6ee504f97b8910a2eb9

### PR DESCRIPTION
**Pull request by SantoJambit**
Resolves #2

### CODE REVIEW

File: tests/assets/defaults.ts [hunk 1]
```diff
@@ -1,2 +1,3 @@
 export { default as one } from './defaults-one';
 export { default as two } from './defaults-two';
+export { default as three } from './defaults-three';
```
Comment (FlorianRappl) [inline tests/assets/defaults.ts]: Could you put this in its own file?

Right now this mixes two tests.

File: tests/defaults.test.ts [hunk 2]
```diff
@@ -10,5 +10,9 @@ test('should handle renaming of default exports', () => {
   export const two: {
     (): string;
   };
+
+  export const three: {
```
Comment (FlorianRappl) [inline tests/defaults.test.ts]: Should be in its own test.

**Comment by SantoJambit** [comment]
@FlorianRappl I found a solution.. not sure if this is the nicest way to do this, but it works.

**Comment by FlorianRappl** [comment]
Looks great!

**Pull request merged**